INSTALL.md

```diff
@@ -50,7 +50,17 @@ Kernel compile flags can usually be checked by looking at `/proc/config.gz` or
 
 ## Ubuntu Xenial - Binary
 
-Only the nightly packages are built for Ubuntu 16.04, but the steps are very straightforward. No need to upgrade the kernel or compile from source!
+The stable and the nightly packages are built for Ubuntu 16.04. The steps are very straightforward, no need to upgrade the kernel or compile from source!
+
+**Stable Packages**
+
+```bash
+echo "deb [trusted=yes] https://repo.iovisor.org/apt/xenial xenial main" | sudo tee /etc/apt/sources.list.d/iovisor.list
```

Comment:
The stable packages are signed, it shouldn't be necessary to have `[trusted=yes]`. The key is at https://repo.iovisor.org/GPG-KEY.

Reply:
@drzaeus77 so is the new version correct?